src/ucp/wireup/wireup.c

```diff
@@ -364,6 +364,13 @@ void ucp_wireup_remote_connected(ucp_ep_h ep)
     }
 
     for (lane = 0; lane < ucp_ep_num_lanes(ep); ++lane) {
+        if (ep->uct_eps[lane] == NULL) {
```

Comment:
Does it mean that we call remote_connected for TMP EP?
I think we should not do it

Reply:
yes, otherwise there is a problem in ucp_wireup_ep_check - from one hand, cannot call ep_check on next_ep if it's not connected, but from other hand, wireup_ep insite tmp_ep was never marked as connected even though it is
that said - I see I tests that doing this creates problems with config_intersect, checking...